katana/code_coverage.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'jacoco'
```

Comment:
Thanks to http://vgaidarji.me/blog/2017/12/20/how-to-configure-jacoco-for-kotlin-and-java-project/